getting-started.md

```diff
@@ -83,6 +83,31 @@ If everything works, you should see your starter app on your device or simulator
   </div>
 </div>
 
+## Viewing source code changes near instantly with 'hot reload'
```

Comment:
I think this would flow slightly better if reordered like this:

1. Line 86, heading
1. Line 96, value statement
1. Line 84, illustration of value statement
1. Lines 88-92 + 98-, remaining content